ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-blacklist/user-blacklist.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+import { hot } from 'react-hot-loader'
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import { keyframes } from '../../styles/styled-components'
+import { Button, buttonTypeEnum } from '../button'
+import BlacklistItemContainer from '../../container/blacklist-item'
+
+const expandAnimation = keyframes`
+  from {
+      transform: scale(0)
+  }
+  to {
+      transform: scale(1)
+  }
+`
+
+const collapseAnimation = keyframes`
+  from {
+    transform: translateY(0) scaleY(1);
+  }
+  to {
+    transform: translateY(-50%) scaleY(0);
+  }
+`
+
+const EmptyBlacklist = styled.div`
+  white-space: normal;
+  padding: ${props => props.theme.minimumSpacing};
+  text-align: center;
+  font-size: ${props => props.theme.largeFontSize};
+  overflow: hidden;
+  animation: ${expandAnimation} ${props => props.theme.coreTransitionTime}
+    linear;
+`
+
+const ItemsWrapper = styled<Props, 'div'>('div')`
+  > *,
+  > * > td {
+    border-top: 1px solid rgba(255, 255, 255, 10%);
```

Comment:
This needs to be the equivalent of this in `less`
```suggestion
    border-top: 1px solid fade(@contrastColor, 10%);
```